src/Func.h

```diff
@@ -853,6 +853,12 @@ class Func {
      * splitting it up. If bounds inference decides that it requires
      * more of this function than the bounds you have stated, a
      * runtime error will occur when you try to run your pipeline.
+     *
+     * The min argument may be an undefined Expr, in which case the
```

Comment:
I can't think of exactly what it should be, but it seems like there should be a bound overload or equivalent that only takes an extent. Using undefined Exprs is only surfaced to users in a few places, and it's easy to confuse with undef().